VirtoCommerce.Platform.Web/Controllers/Api/AssetsController.cs

```diff
@@ -117,12 +121,29 @@ public AssetsController(IBlobStorageProvider blobProvider, IBlobUrlResolver urlR
                 await Request.Content.ReadAsMultipartAsync(blobMultipartProvider);
 
                 foreach (var blobInfo in blobMultipartProvider.BlobInfos)
-                {  
+                {
                     blobInfo.RelativeUrl = blobInfo.Key;
                     blobInfo.Url = _urlResolver.GetAbsoluteUrl(blobInfo.Key);
                     retVal.Add(blobInfo);
                 }
             }
+
```

Comment:
We decided to do not create AssetEntry on this commands. Su just needs to remove this logic from here.

Reply:
@eMazeika may you fix today?

Reply:
all done